spec/support/capybara_actions.rb

```diff
@@ -7,7 +7,9 @@ def blur_from(locator)
   end
 
   def within_section(title, &)
-    within(:xpath, "//section[contains(header/text(),'#{title}')]", &)
+    within(:xpath, "//section[contains(h2/text(),'#{title}')]", &)
+  rescue Capybara::ElementNotFound
```

Comment:
This may not be the best approach, but I could not get a value xpath OR query to work, and `within` raises an error when not found, so we can't do something like `within(..) || within(..)`... it also complained if I passed a string in (h2 or h3) and used string interpolation.. 

I'm open to suggestion if this seems problematic.